backend/src/main/java/ch/puzzle/okr/controller/MeasureController.java

```diff
@@ -53,4 +54,17 @@ public ResponseEntity<Object> createMeasure(@Valid @RequestBody MeasureDto measu
         }
     }
 
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "Saved new measure to db",
```

Comment:
Ist keine neue Measure. Beschreibung anpassen

Reply:
Description gewechselt auf "Updated measure in db"